libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -204,6 +205,12 @@ private static PolicyManager createPolicyManager() {
                         new LoadNativeLibrariesEntitlement(),
                         new FilesEntitlement(List.of(FileData.ofRelativePath(Path.of(""), FilesEntitlement.BaseDir.DATA, READ_WRITE)))
                     )
+                ),
+                new Scope(
```

Comment:
Could we only add this if the trust store property is present? Then we don't need to handle an empty list of files.

Reply:
Initially did it this way to minimize the risk of conflicts, but changed to add the scope conditionally 👍

Reply:
There will be conflicts anyway, this class is getting hammered :)